src/lib/include/libsed.h

```diff
@@ -117,8 +117,15 @@ struct sed_blocksid_supported_feat {
 
 struct sed_opal_level0_discovery {
 	struct {
+		uint64_t feat_tper:1;
+		uint64_t feat_locking:1;
+		uint64_t feat_geometry:1;
+		uint64_t feat_datastr_table:1;
+		uint64_t feat_opalv100:1;
+		uint64_t feat_opalv200:1;
 		uint64_t feat_blocksid:1;
-		uint64_t reserved:63;
+		uint64_t feat_sum:1;
+		uint64_t reserved:56;
```

Comment:
shouldn't it be reserved:55

Reply:
8 BITS for the feature and 56 BITS reserved (56 + 8 = 64)

Reply:
Oh I see cnl bit is added in later commit

Reply:
yes. and the reserved is changed to 55 in that commit as well